src/tools/synthetic_node.rs

```diff
@@ -460,7 +460,7 @@ impl Default for MessageCodec {
                 .length_field_offset(16)
                 .little_endian()
                 .num_skip(0)
-                .max_frame_length(65536) // FIXME
+                .max_frame_length(1048576)
```

Comment:
we could leave a small comment here so it's not forgotten

Reply:
Comment left.
Don't forget there can be also `git blame` used to check why the line looks like that 😸